staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -609,6 +609,8 @@ type VirtualMachineInstanceMigrationState struct {
 	// +nullable
 	EndTimestamp *metav1.Time `json:"endTimestamp,omitempty"`
 
+	// The timestamp at which the target node detects the domain is active
+	TargetNodeDomainStartTimestamp *metav1.Time `json:"targetNodeDomainStartTimestamp,omitempty"`
```

Comment:
Domain was never stopped I think we should use Ready instead of DomainStart, also we have to talk about migration here and the target is not a node.
```suggestion
	// The timestamp at which the target node detects the domain is active
	MigrationTargetReadyTimestamp *metav1.Time `json:"migrationTargetReadyTimestamp,omitempty"`
```

Reply:
I changed this to `targetNodeDomainReadyTimestamp`. I know that's slightly different than your suggestion, but it aligns more closely with the existing targetNodeDomainDetected value